instrumentation/nocode/src/test/config/nocode.yml

```diff
@@ -0,0 +1,24 @@
+- class: com.splunk.opentelemetry.instrumentation.nocode.NocodeInstrumentationTest$SampleClass
+  method: doSomething
+  spanName: "this.getName()"
+  attributes:
+    - key: "details"
+      value: this.getDetails()
+    - key: "map.size"
+      value: this.getMap().entrySet().size()
+
+- class: com.splunk.opentelemetry.instrumentation.nocode.NocodeInstrumentationTest$SampleClass
+  method: throwException
+  spanKind: SERVER
+  attributes:
+    - key: "five"
+      value: param0.toString().substring(0)
+
+- class: com.splunk.opentelemetry.instrumentation.nocode.NocodeInstrumentationTest$SampleClass
+  method: doInvalidRule
+  spanName: "this.thereIsNoSuchMethod()"
+  spanKind: INVALID
+  attributes:
+    - key: "notpresent"
+      value: "invalid.noSuchStatement()"
+
```

Comment:
It would be cool if we could be consistent in our use or lack of quotes. I'd prefer to drop them in all places where they're not needed (one of the few benefits if yaml, heh)

Reply:
I'm specifically testing this behavior of yaml for our intended use!